dockerfiles/run/guice/memory/destination/conf/elasticsearch.properties

```diff
@@ -17,6 +17,8 @@
 
 #  This template file can be used as example for James Server configuration
 #  DO NOT USE IT AS SUCH AND ADAPT IT TO YOUR NEEDS
+# Read https://james.apache.org/server/config-elasticsearch.html for further details (metrics only)
```

Comment:
It's not really related to this PR but it seems strange to change value in an `elasticsearch.properties` file for a memory product.